src/withElementPortal.js

```diff
@@ -5,21 +5,18 @@ function getDisplayName(Component) {
   return Component.displayName || Component.name || 'Component';
 }
 
-const withElementPortal = (Child) => {
-  const Portal = ({ id, selector, shouldReset, mapDomNodeToProps, ...childProps }) => {
-    const ChildWrapper = (props) => <Child {...childProps} {...props} />;
-    return <ElementPortal
-      id={id}
-      selector={selector}
-      mapDomNodeToProps={mapDomNodeToProps}
-      shouldReset={shouldReset}
-      view={ChildWrapper}
-    />;
-  };
+const withElementPortal = (portalProps) => (component) => {
```

Comment:
@vitorbal tried to get me to change this a while back, but there didn't seem to be a great reason for it. The new `mapNodeToProps` callback gives this a good reason though, so I'm cool with the change.